app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,93 @@
+image_2_alt: "",
+) do %>
+<p>Computing focuses on 3 areas:</p>
+<ul>
+<li>information and communication technology</li>
+<li>digital literacy</li>
+<li>computer science</li>
+</ul>
+<p>You'll inspire pupils to access new fields and teach them new, transferable skills that will open lots of sectors, from engineering to healthcare, and could lead to careers such as software engineering and web design.</p>
+<% end %>
+</section>
+<section class="col col-720">
+<h2>What you'll be teaching</h2>
+<p>You'll teach themes from the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-computing-programmes-of-study/national-curriculum-in-england-computing-programmes-of-study">national curriculum for computing</a>, with opportunities to develop your pupils' computational thinking.</p>
+<p>When teaching 11 to 14 year olds (key stage 3), you could use interactive exercises during your lessons, like creating a robot car. Themes you might cover include:</p>
+<ul class="vertical-tags">
+<li class="tag">Algorithms</li>
+<li class="tag">Programming and debugging</li>
+<li class="tag">Creative projects</li>
+<li class="tag">Use of logical reasoning</li>
+<li class="tag">Computer systems</li>
+<li class="tag">Uses of technology</li>
+<li class="tag">Online safety </li>
+</ul>
+<p>You'll teach 14 to 16 year olds (key stage 4) computer science as a GCSE qualification, teaching pupils ways to protect their online privacy and identity. Themes you might also cover include:</p>
+<ul class="vertical-tags">
+<li class="tag">Computational thinking</li>
+<li class="tag">Digital media and information technology</li>
+<li class="tag">Computer networks</li>
+<li class="tag">Cyber security</li>
+<li class="tag">Data representation</li>
+<li class="tag">Digital technology</li>
+</ul>
+<p>For 16 to 18 year olds (key stage 5), you'll teach computer science as an A level qualification, helping pupils to develop websites and prepare them for further learning or employment. Themes you might also cover include:</p>
+<ul class="vertical-tags">
+<li class="tag">Abstraction</li>
+<li class="tag">Writing programmes to solve problems</li>
+<li class="tag">Efficiency of algorithms</li>
+<li class="tag">Data structures and the use of databases</li>
+<li class="tag">Systems architecture</li>
+<li class="tag">Computing related mathematics</li>
+</ul>
+
+<p><a href="https://www.thenational.academy/teachers/programmes/computing-secondary-ks4-l/units">Explore what teaching a computer lesson would be like</a>.</p>
+</section>
+<section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "Every lesson presents an opportunity to delve into the latest advancements in the field and inspire students to become innovators. Teaching computing is about fostering a deep understanding of technology while enabling students to develop the skills and resilience to thrive in this digital age.",
+    name: "Shiulee Begum, Curriculum Lead Computing and Business",
```

Comment:
```suggestion
    name: "Shiulee Begum, curriculum lead, computing and business",
```
@caitlin-reidy As before - shouldn't we remove caps from titles and subjects? (plus add a comma)